operator/v1/types_ingress.go

```diff
@@ -1399,6 +1399,132 @@ type IngressControllerHTTPHeaders struct {
 	// +nullable
 	// +optional
 	HeaderNameCaseAdjustments []IngressControllerHTTPHeaderNameCaseAdjustment `json:"headerNameCaseAdjustments,omitempty"`
+
+	// actions specifies options for performing action on headers.
+	// Note that this option only applies to cleartext HTTP connections
+	// and to secure HTTP connections for which the ingress controller
+	// terminates encryption (that is, edge-terminated or reencrypt
+	// connections).  Headers cannot be set or deleted for TLS passthrough
+	// connections.
+	// Setting HSTS header is supported via another mechanism in Ingress.Spec.RequiredHSTSPolicies.
```

Comment:
```suggestion
	// Setting the HSTS (`Strict-Transport-Security`) header is not supported via actions. `Strict-Transport-Security ` may only be configured in Ingress.Spec.RequiredHSTSPolicies.
```

Reply:
Strictly speaking, it's the `haproxy.router.openshift.io/hsts_header` annotation that can be used to specify the HSTS header, but we can reference `RequiredHSTSPolicies` here as its godoc in turn references the annotation.

Reply:
I don't have a strong opinion on which is referenced, I was just rewording what was there, do you have a preference?

Reply:
How about this?
```suggestion
	// Setting the HSTS (`Strict-Transport-Security`) header is not supported via actions. `Strict-Transport-Security ` may only be configured using the "haproxy.router.openshift.io/hsts_header" route annotation, and only in accordance with the policy specified in Ingress.Spec.RequiredHSTSPolicies.
```

Reply:
done

Reply:
+1 from me, thanks both